src/test/skript/tests/syntaxes/expressions/ExprNewBannerPattern.sk

```diff
@@ -0,0 +1,26 @@
+test "new banner pattern":
```

Comment:
needs tests for invalid states

Reply:
invalid states as in?

Reply:
nulls, wrong types, wrong blocks, wrong items, etc